sdcm/stress_thread.py

```diff
@@ -110,10 +112,15 @@ def create_stress_cmd(self, node, keyspace_idx):
         elif 'keyspace=' not in stress_cmd:  # if keyspace is defined in the command respect that
             stress_cmd = stress_cmd.replace(" -schema ", " -schema keyspace=keyspace{} ".format(keyspace_idx))
 
+        if self.sla_role and '<role ' in stress_cmd:
```

Comment:
Also we should make sure we don't add multiple user/password flags

Reply:
added validation